.github/ISSUE_TEMPLATE/new_chapter.yml

```diff
@@ -0,0 +1,49 @@
+name: "\U0001F4D6 New Chapter Template 2"
+description: Template for proposing a new chapter
+title: "<title>"
+labels: book
+body:
+- type: markdown
+  attributes:
+    value: "Please complete the following sections when you want to propose a new chapter. If you have write access to the repository, please also assign the appropriate label (or labels) to your issue."
+- type: textarea
+  id: summary
+  attributes:
+    label: Summary of proposed chapter
+    description: |
+      Please provide a detailed description of the chapter you are writing or proposing. If you have started writing the chapter, also provide details about the pull request you've made as well as the branch you've been working from. Please provide as much context as possible and link to other related issues and pull requests. Consider the following questions:
```

Comment:
Following up on meeting with @aldenc – is there a way of hacking the size of this text so that it's not so small? Maybe a question for @sgibson91?

Reply:
I don't think so 😕 unless you make it not part of the `textarea` block and add a `markdown` block above

Reply:
Seems like it's okay then as is, thanks @sgibson91 ! 😄